lib/agent0/agent0/hyperdrive/crash_report/crash_report.py

```diff
@@ -236,6 +241,11 @@ def log_hyperdrive_crash_report(
 
     logging.log(log_level, logging_crash_report)
 
+    if log_to_rollbar:
+        log_rollbar_exception(trade_result.exception, log_level, json.loads(logging_crash_report))
```

Comment:
I would log the crash report that's written to file, this logging crash report is a subset to avoid blowing up stdout

Reply:
i.e., the log in the if statement below